.github/workflows/LocalesTest.yml

```diff
@@ -0,0 +1,65 @@
+name: Test Different Locales
+
+on:
+  push:
+    branches: [ v9.0 ]
+  pull_request:
+    branches: [ v9.0 ]
+  schedule:
+    - cron: 0 6 * * 1
```

Comment:
Do we actually need to run this on a schedule? I suspect we only care to run this when there are changes.

Reply:
Just for consistency with our build and test action, but no probably unecessary.

Reply:
I would posit that neither needs a cron trigger